talentlink-connector/processes/rest/Position.p.json

```diff
@@ -0,0 +1,159 @@
+{
+  "format" : "10.0.0",
+  "id" : "184C3626F42D7C7C",
+  "kind" : "CALLABLE_SUB",
+  "config" : {
+    "data" : "talent.link.Data"
+  },
+  "elements" : [ {
+      "id" : "f0",
+      "type" : "CallSubStart",
+      "name" : "getPositionById(Long)",
+      "config" : {
+        "callSignature" : "getPositionById",
+        "input" : {
+          "params" : [
+            { "name" : "positionId", "type" : "Long" }
+          ],
+          "map" : {
+            "out.positionId" : "param.positionId"
+          }
+        },
+        "result" : {
+          "params" : [
+            { "name" : "positions", "type" : "java.util.List<com.lumessetalentlink.shared.apiproxy.tlk.rest.client.OpeningFullDto>" },
+            { "name" : "error", "type" : "ch.ivyteam.ivy.bpm.error.BpmError" }
+          ],
+          "map" : {
+            "result.positions" : "in.positions",
+            "result.error" : "in.#error"
+          }
+        },
+        "tags" : "connector"
+      },
+      "visual" : {
+        "at" : { "x" : 96, "y" : 64 }
+      },
+      "connect" : { "id" : "f4", "to" : "f3" }
+    }, {
+      "id" : "f1",
+      "type" : "CallSubEnd",
+      "visual" : {
+        "at" : { "x" : 352, "y" : 64 }
+      }
+    }, {
+      "id" : "f3",
+      "type" : "RestClientCall",
+      "name" : "Position",
+      "config" : {
+        "path" : "/position",
+        "clientId" : "d8e4652e-41d5-4023-90c5-5c822b8600e2",
+        "clientErrorCode" : "ivy:error:rest:client",
+        "queryParams" : {
+          "query" : "java.net.URLEncoder.encode(\"{positions(first: 10, after: 0, where: {id_eq: \"+in.positionId+\"}) {id, expectedContractDates{start, end}, location{city, region, country}, configurableFields{lovs{valueLovId, value} }, organization{id, name, parentId, level, structure{id, name, level, parentId}}}}\", \"utf-8\")",
```

Comment:
Sad that these parameters are not correctly stated in the openapi-descriptor, so you would not have to describe it manually and encode it here.

What source did you use to generate the rest-client? It looks to me as if you had a swagger-2.0 definition. Is it public available anywhere? Could it be that talentlink also provides an openapi-3.0 spec? ... so we would very likely gain this query-parameter handling for free ... :thinking:

Reply:
Yes, I see, unfortunately they did not describe the structure of this query object correctly ... or not as good as they can. 
It's to keep it as is then.